buildconfig/config.py

```diff
@@ -165,6 +166,10 @@ def main(auto=False):
 Only SDL2 is supported now.""")
 
     kwds = {}
+
+    if "docs" in sys.argv:
```

Comment:
```suggestion
    if sys.argv[1] == "docs":
```
I would like this to be more stricter

Reply:
Committed